### PR TITLE
Add release notes for v4.0.16.

### PR DESCRIPTION
**Pull request by TTVXStarWake**
Add release notes for v4.0.16.

### CODE REVIEW

File: site/pages/docs/versions/rdmp-en.hbs [hunk 1]
```diff
@@ -20,12 +20,12 @@
...
-				<td><time>2015-08-18</time></td>
-				<td>v4.0.17 release</td>
+				<td><time>2015-09-18</time></td>
+				<td>v4.0.18 release</td>
 			</tr>
 			<tr>
-				<td><time>2015-07-21</time></td>
-				<td>v4.0.16 release</td>
+				<td><time>2015-08-18</time></td>
+				<td>v4.0.17 release</td>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/rdmp-en.hbs]: Those dates are not the right ones, use the following ones instead

https://github.com/wet-boew/wet-boew/milestones

**Comment by LaurentGoderre** [comment]
You are missing step 2 of the release process. I gues it's not really cleasr but you need to copy the v4.0.15 files and create new ones for v4.0.16

**Comment by MaximMartel** [comment]
i forgot to git add the file since they were new. Added it in the last commit.

**Comment by LaurentGoderre** [comment]
In the future, this should be on a seperate branch not on master. Can you squash the two commit using `git rebase -i HEAD~2` and update your PR using `git push -f origin master`

Note that updating your PR is the only time you should force push.

**Comment by LaurentGoderre** [comment]
You should update the dates as well sop that there is only one commit

**Pull request merged**